### PR TITLE
Cleanup before merge

### LINKED ISSUE #14: Cleanup repo before merge
We want to preserve git history on panoptes/POCS#98 but first need to do some cleanup and fix potentially conflicting files.

### PR DESCRIPTION
**Pull request by wtgee**
Pre merge cleanup, closes #14

### CODE REVIEW

File: scripts/scrub_mongo.py [hunk 1]
```diff
@@ -1,25 +0,0 @@
-#!/usr/bin/env python
```
Comment (jamessynge) [inline scripts/scrub_mongo.py:1]: Why is this file removed? Obsolete?

File: setup.py [hunk 2]
```diff
@@ -1,56 +0,0 @@
-#!/usr/bin/env python
```
Comment (jamessynge) [inline setup.py:1]: I would have expected this to be a rename (e.g. to setup_peas.py), so that it becomes easier to see the merge of this file's contents in POCS.

File: scripts/scrub_mongo.py (hunk 1 above)
Comment (wtgee) [reply scripts/scrub_mongo.py:1]: Yes, that's a very old file. We will keep this repo around as archive and reference so I saw no reason to move it.

File: setup.py (hunk 2 above)
Comment (wtgee) [reply setup.py:1]: The `setup.py` works via `python setup.py [install|develop]` and is a standard python name (I think?). The one file will install both `peas` and `pocs` when run based on how it is set up.
